CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 ^^^^^^^^^^^^^^
 
+version 1.2.1
+**************
+- Updated axe.min.js to latest version of axe-core
```

Comment:
Do you have the version of axe-core that we're using noted in the docs?

Reply:
I do not. I can do that 👍